tests/backup/backup_restore_basic_test.go

```diff
@@ -895,3 +896,197 @@ var _ = Describe("{CustomResourceRestore}", func() {
 
 	})
 })
+
+// BackupSyncBasicTest take a good number of backups check if backup sync is working
+var _ = Describe("{BackupSyncBasicTest}", func() {
+	numberOfBackups, _ := strconv.Atoi(getEnv(maxBackupsToBeCreated, "10"))
+	timeBetweenConsecutiveBackups := 4 * time.Second
+	backupNames := make([]string, 0)
+	numberOfSimultaneousBackups := 20
+	var contexts []*scheduler.Context
+	labelSelectors := make(map[string]string)
+	var backupLocationUID string
+	var cloudCredUID string
+	var backupName string
+	var cloudCredUidList []string
+	var appContexts []*scheduler.Context
+	var bkpNamespaces []string
+	var clusterUid string
+	var clusterStatus api.ClusterInfo_StatusInfo_Status
+	var customBackupLocationName string
+	var credName string
+	bkpNamespaces = make([]string, 0)
+	backupNamespaceMap := make(map[string]string)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("BackupSyncBasicTest",
+			"validate that the backup sync syncs all the backups present in bucket", nil, 58040)
+		log.InfoD("Deploy applications")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+	It("validate that the backup sync syncs all the backups present in bucket", func() {
```

Comment:
nit: Can you make `v` in `validate` upper case?

Reply:
Done